src/index.js

```diff
@@ -22,16 +26,33 @@ window.searchForecast = () => {
       key: '5086cdd8745d497fa07f6786dafbbcdc'
     }
     createForecast(params);
-
   })
 };
 
 function createForecast(params) {
   axios.get('https://api.weatherbit.io/v2.0/forecast/daily', { params })
```

Comment:
When you call this function only `lat` and `lon` can be different, key always remains the same
So you can better define it here to omit repetitions
```suggestion
  axios.get('https://api.weatherbit.io/v2.0/forecast/daily', {
    params: {
       ...params,
       key: API_KEY
    }
  })
```